### PR TITLE
feature/v1-046/Learning course - video

### PR DESCRIPTION
**Pull request by Julia-Maliutsina**
Implemented:
1. React video player with custom play button.
2. Function to define type of material: text or video.
3. API connection: mock course material replaced with data from server.

### CODE REVIEW

File: frontend/src/pages/LearningCourse/LearningCourseContainer.tsx [hunk 1]
```diff
@@ -14,6 +17,11 @@ const LearningCourseContainer: React.FC = () => {
   const [backDisabled, setBackDisabled] = useState(true);
   const [forwardDisabled, setForwardDisabled] = useState(false);
 
+  const params = useParams();
+  const { data } = useGetClientCourseInfo(params.courseId);
+
+  const maxStage = data ? data.course.materials.length : 1;
```
Comment (hauteballe) [inline frontend/src/pages/LearningCourse/LearningCourseContainer.tsx:23]: Please remove magic number

**Review by hauteballe — CHANGES_REQUESTED** [review]

**Review by hauteballe — DISMISSED** [review]

File: frontend/src/pages/LearningCourse/LearningCourseContainer.tsx [hunk 2]
```diff
@@ -25,34 +35,49 @@ const LearningCourseContainer: React.FC = () => {
...
-      courseDescription={COURSE_DESCRIPTION}
+      courseDescription={courseDescription}
       testEnabled={testEnabled}
       backDisabled={backDisabled}
       forwardDisabled={forwardDisabled}
+      material={material}
+      materialType={materialType}
     />
-  );
+  ) : null;
```
Comment (k-krasilnikova) [inline frontend/src/pages/LearningCourse/LearningCourseContainer.tsx:80]: Remove please `null` We shouldn't allow such logic

File: frontend/src/pages/LearningCourse/LearningCourse.tsx [hunk 3]
```diff
@@ -26,63 +30,76 @@ interface LearningProps {
...
-          </NextButtonWrapper>
-        )}
-      </LearningWrapper>
-    </LearningPageContainer>
-  </AuthorizedLayout>
-);
+  material,
+  materialType,
+}) => {
+  return (
```
Comment (k-krasilnikova) [inline frontend/src/pages/LearningCourse/LearningCourse.tsx:53]: It's an arrow function!
Remove please `return` statement

File: frontend/src/api/myCourses/getMyCourseInfo.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import { useQuery, UseQueryResult } from 'react-query';
+import { AxiosError } from 'axios';
+
+import { apiClientWrapper } from 'api/base';
+import { API } from 'constants/routes';
+import { REQUEST_ERRORS } from 'constants/authConstants';
+import { ClientCourse } from 'types/clientCourse';
+
+const useGetClientCourseInfo = (
+  _id: string | undefined,
```
Comment (k-krasilnikova) [inline frontend/src/api/myCourses/getMyCourseInfo.ts:10]:
```suggestion
  _id?: string,
```

**Review by k-krasilnikova — CHANGES_REQUESTED** [review]

**Review by k-krasilnikova — APPROVED** [review]

**Pull request merged**